src/App.js

```diff
@@ -1,16 +1,97 @@
-import React from 'react';
+import React  from 'react';
+import './App.css';
+import TodoList from './components/TodoComponents/TodoList';
+let generateId = Date.now();
+const TodoItems = [
+  {
+    task: 'Organize Garage',
+    id: generateId,
+    completed: true
+  },
+  {
+    task: 'Bake Cookies',
+    id: generateId+1,
```

Comment:
Instead of doing `generateId + 1` I think writing a function to do it for you would have been a good idea.